tests/pipeline/test_examples.py

```diff
@@ -1,10 +1,14 @@
 import importlib
+import os
+import sys
 
 import pytest
 
 import tests.utils as utils
 
-dot_path_to_addon = utils.get_dot_path_from_tests_to_current_dir(__file__)
+sys.path.append(os.path.abspath(f"examples/{utils.get_path_from_tests_to_current_dir(__file__)}"))
```

Comment:
Why is this line added? It is not necessary since all examples are imported with an `examples.{dot_path_to_addon}` prefix.
Also it won't work on Windows.

Reply:
Please, check workflow runs - it works on Windows.
It _is_ necessary, since there are relative imports in the imported examples.

Reply:
This is necessary because of  `from _pipeline_utils import SCRIPT, should_auto_execute, auto_run_pipeline
`

Reply:
Pls, add `TODO` to remove when we will add example utils into pipy package